server/src-lib/Hasura/SQL/Types.hs

```diff
@@ -80,6 +80,10 @@ infixr 6 <<>
   T.dquote (dquoteTxt a) <> rTxt
 {-# INLINE (<<>) #-}
 
+fromL :: (DQuote a) => [a] -> T.Text
```

Comment:
Needs better naming. `fromL` doesn't say anything about the function, nor the type signature.